ios/trainer/Info.plist

```diff
@@ -2,6 +2,14 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSExceptionDomains</key>
```

Comment:
Can you give me a high level explanation of the changes in this file?

Reply:
this is basically our main config file for our ios app. we're allowing insecure connections from localhost here, so we can hit http://localhost:5000 for local testing, as opposed to https: